SIT.Manager.Avalonia/ViewModels/PlayPageViewModel.cs

```diff
@@ -43,8 +44,14 @@ public partial class PlayPageViewModel : ViewModelBase
         [ObservableProperty]
         private bool _rememberMe;
 
+        [ObservableProperty]
+        private ManagerConfig _managerConfig;
+
         [ObservableProperty]
         private string _quickPlayText = "Start Server and Connect";
+        
+        [ObservableProperty]
+        private ManagerConfig _managerConfig;
```

Comment:
I just downloaded the branch and ran it. Everything seemed to work right even following the steps you did in the video it behaved properly.

Only thing I had to do was delete one of these as you have the same thing defined twice now